workflow/rules/mapping.smk

```diff
@@ -42,12 +42,13 @@ rule mark_duplicates:
     params:
         extra=lambda wc: "{c} {b} {d}".format(
             c=config["params"]["picard"]["MarkDuplicates"],
-            b="" if units.loc[wc.sample]["umis"].isnull().any() else "BARCODE_TAG=RX",
-            d="TAG_DUPLICATE_SET_MEMBERS=true"
+            b=""
```

Comment:
I'd suggest moving that to a regular python function in common.smk

Reply:
Done. Tests are running... :runner: